.circleci/config.yml

```diff
@@ -90,7 +90,24 @@ jobs:
             sudo docker login -u redisfab -p $DOCKER_REDISFAB_PWD
             make -C opt/build/docker build
             sudo make -C opt/build/docker publish
-
+  build-gpu:
+    machine:
+      docker_layer_caching: true
+      resource_class: gpu.nvidia.medium
+      image: ubuntu-1604-cuda-10.1:201909-23
+    steps:
+      - checkout
+      - run:
+          name: Bulding Docker
+          command: |
+            docker build -f Dockerfile.gpu-test -t redisai-gpu-test .
+      - run:
+          name: Test
+          command: |
+            mkdir -p ~/workspace/tests
+            docker run --gpus all -v $HOME/workspace/tests:/data/tests/logs -it --rm redisai-gpu-test
```

Comment:
I'm not sure whether CircleCI supports volumes (-v) with docker.

Reply:
Just verified. It works on `machine` executor